clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -407,8 +407,6 @@ public void synchronizeKeyToPassive(PassiveSynchronizationChannel<EhcacheEntityM
 
   @Override
   public void createNew() {
-    management.init();
```

Comment:
because the passive does not have the loadExisting method. 
The way platform works now works fine for the createNew() so I didn't change the passive entities.
This workaround will be removed once I'll have done https://github.com/Terracotta-OSS/terracotta-platform/issues/256